demos/link-prediction/hinsage/movielens-recommender.ipynb

```diff
@@ -469,17 +469,18 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "150/150 [==============================] - 5s 35ms/step - loss: 12.0534 - root_mean_square_error: 3.4710 - mean_absolute_error: 3.2834\n",
+      "  ['...']\n",
+      "150/150 [==============================] - 25s 165ms/step - loss: 11.8834 - root_mean_square_error: 3.4467 - mean_absolute_error: 3.256210s - loss: 11.8462 - root_mean_square_erro\n",
       "Untrained model's Test Evaluation:\n",
-      "\tloss: 12.0534\n",
-      "\troot_mean_square_error: 3.4710\n",
-      "\tmean_absolute_error: 3.2834\n"
+      "\tloss: 11.8834\n",
+      "\troot_mean_square_error: 3.4467\n",
+      "\tmean_absolute_error: 3.2562\n"
      ]
     }
    ],
    "source": [
     "test_metrics = model.evaluate_generator(\n",
-    "    test_gen, verbose=1, use_multiprocessing=True, workers=num_workers\n",
+    "    test_gen, verbose=1, use_multiprocessing=False, workers=num_workers\n",
```

Comment:
We can get rid of the `workers` param here 'cause we're not using multiprocessing.

```suggestion
    "    test_gen, verbose=1, use_multiprocessing=False\n",
```

Reply:
I think the `workers` param should get used to do multithreading instead - see https://github.com/stellargraph/stellargraph/pull/925#discussion_r383086045

Reply:
oh good point, missed that in the docs 👍